uber/templates/base.html

```diff
@@ -29,6 +29,12 @@
 
         <script src="../static/deps/combined.min.js" type="text/javascript"></script>
         <script src="../static/js/common-static.js" type="text/javascript"></script>
+        <script type="text/javascript">
+            var csrf_token = '{{ c.CSRF_TOKEN }}';
```

Comment:
Aren't we already doing this somewhere else?

Reply:
Whoops, yeah, just a few lines below this...

Reply:
Dang, I thought that might be a merge error! I fix!